ansible/roles/build_tools/README.md

```diff
@@ -16,4 +16,7 @@ sudo apt-get update
 
 # Install ccache
 sudo apt-get install -y ccache
+
+# Add ccache to PATH
+echo 'export PATH="/usr/lib/ccache/:$PATH"' >> ~/.bashrc
 ```
```

Comment:
@mebasoglu Hello! Thank you so much for working on this ccache issue! Could you please consider to add following lines for handling cases when the cached one exists or not? :pray:

[Reference](https://github.com/autowarefoundation/autoware/issues/4605)

If other files are also needed to be modified, or following lines contains potential side effects, your feedback is really  appreciated. Thank you in advance!

```
echo ""                                                                                   >> ~/.bashrc
echo '# Set /var/tmp/ccache if exists otherwise set nothing'                              >> ~/.bashrc
echo 'export CCACHE_DIR=$( [ -f /var/tmp/ccache ] && echo "/var/tmp/ccache" || echo "" )' >> ~/.bashrc
echo ""                                                                                   >> ~/.bashrc

echo ""                                                                                     >> ~/.bashrc
echo '# Set /usr/lib/ccache/gcc if exists otherwise set the output of `which gcc`'          >> ~/.bashrc
echo 'export CC=$( [ -f /usr/lib/ccache/gcc ] && echo "/usr/lib/ccache/gcc" || which gcc )' >> ~/.bashrc
echo ""                                                                                     >> ~/.bashrc

echo ""                                                                                      >> ~/.bashrc
echo '# Set /usr/lib/ccache/g++ if exists otherwise set the output of `which g++`'           >> ~/.bashrc
echo 'export CXX=$( [ -f /usr/lib/ccache/g++ ] && echo "/usr/lib/ccache/g++" || which g++ )' >> ~/.bashrc
echo ""                                                                                      >> ~/.bashrc
```